### PR TITLE
feat(filter-opd): include the null 'AsalNaskah'

### PR DESCRIPTION
**Pull request by samudra-ajri**
## Overview
- Add the condition where the AsalNaskah is null.
- Inbox which has null AsalNaskah will be provided on the result.

## Evidence
title: feat(filter-opd): include the null 'AsalNaskah'
project: sidebar
participants: @samudra-ajri @indraprasetya154 @fajarhikmal214 @azophy

### CODE REVIEW

**Review by indraprasetya154 — APPROVED** [review]
LGTM

**Review by fajarhikmal214 — APPROVED** [review]

File: src/app/Http/Traits/InboxFilterTrait.php [hunk 1]
```diff
@@ -346,8 +346,8 @@ private function filterBySenderDepts($query, $filter)
                         ->where(fn ($query)
                         => $query->whereIn('role.roleCode', $arrayIds)
                             ->where(fn ($query)
-                            => $query->where('AsalNaskah', 'internal')
-                                ->orWhere('Pengirim', 'internal'))
+                            => $query->whereNull('AsalNaskah')
+                                ->orWhere('AsalNaskah', '!=', 'eksternal'))
```
Comment (azophy) [inline src/app/Http/Traits/InboxFilterTrait.php:350]: eh kalau tidak sama dengan di query build laravel bisa pakai `!=` ya? kalau swl biasanya pakai `<>` gak sih?

contoh di docs laravel:

Comment (samudra-ajri) [reply src/app/Http/Traits/InboxFilterTrait.php]: Betul, Mas.. bisa pakai `!=`. Tapi klw di documentation pakainya `<>`, ya?
Saya sesuaikan di sini https://github.com/jabardigitalservice/office-services/pull/593/commits/10cc0c1a0baef6c5fc240bbae9466785acabddc5 ya, Mas.. 😁

**Review by azophy — APPROVED** [review]
ntaps2 jri

**Pull request merged**